modules/pam_limits/pam_limits.c

```diff
@@ -550,7 +550,6 @@ process_limit (const pam_handle_t *pamh, int source, const char *lim_type,
     int int_value = 0;
     rlim_t rlimit_value = 0;
     char *endptr;
-    const char *value_orig = lim_value;
```

Comment:
Could you move the cleanup part of this change into a separate commit, please?

Reply:
I don't see the point to split 3-lines commit into two: neither first to remove unnecessary variable but leave buggy line, nor first to fix buggy line with unnecessary variable and second to remove it.

Reply:
The point is that unrelated changes that address different issues should be committed separately,